packages/server/src/routes/otherRoutes.ts

```diff
@@ -14,6 +14,14 @@ const defineOtherRoutes = function (app: KoaJsonRpc, relay: Relay, logger: pino.
   app.useRpcRegex(/^engine_.*$/, async () => {
     return logAndHandleResponse('engine', [], () => predefined.UNSUPPORTED_METHOD, app, logger);
   });
+
+  app.useRpcRegex(/^trace_.*$/, async () => {
+    return logAndHandleResponse('trace', [], () => predefined.NOT_YET_IMPLEMENTED, app, logger);
```

Comment:
q: do we want to use the actual name that was received?

Reply:
You mean as in returning `trace_transaction not yet implemented`?